website/static/js/addProjectPlugin.js

```diff
@@ -34,17 +34,31 @@ var AddProject = {
         self.newProjectName = m.prop('');
         self.newProjectDesc = m.prop('');
         self.newProjectCategory = m.prop(self.defaultCat);
+        self.newProjectTemplate = m.prop('');
         self.goToProjectLink = m.prop('');
         self.saveResult = m.prop({});
         self.errorMessageType = m.prop('unknown');
         self.errorMessage = {
             'unknown' : 'There was an unknown error. Please try again later.'
         };
+        self.userProjects =  []; // User nodes
+
         // Validation
         self.isValid = m.prop(false);
         self.checkValid = function _checkValid() {
             self.isValid(self.newProjectName().trim().length > 0);
         };
+
+        var url = $osf.apiV2Url('users/me/nodes/', {query : {'page[size]': 1000}});
```

Comment:
We already make this request on both the dashboard and the My Projects pages. Can we re-use that data, so that we don't need to make this call again?